Doberman/utils.py

```diff
@@ -29,16 +29,22 @@ def find_plugin(name, path):
     :returns constructor: the constructor of the requested device
     """
     strip = False
+    split = False
```

Comment:
This has got a bit messy with all the binary variables and then ifs below. Maybe make one variable be the plugin name and then save that the first time you got through the possibilities, to get rid of the second if?